.github/workflows/lint.yml

```diff
@@ -20,3 +20,5 @@ jobs:
         with:
           python-version: "3.x"
       - uses: pre-commit/action@v3.0.0
+      - run: pip install black
+      - run: black Tools/cases_generator
```

Comment:
Sorry, that's not what I meant. I don't want black to run in CI. I just wanted you to run black after fixing those mypy issues. I'm not ready to run black on every commit.

Reply:
Yeah, and there was some issue with CI so I rollbacked :)